modules/admin_manual/pages/release_notes.adoc

```diff
@@ -560,6 +560,44 @@ Version 10.1 delivers all the prerequisites to be compatible with the Microsoft
 
 ownCloud Server 10.1 introduces WebDAV Locks that allow clients to lock and unlock resources to prevent other users from making changes. The feature has been implemented as a prerequisite for manual file locking and MS Office Online Server compatibility. In the current state, file locking is only available via API. Users can recognize locked files via the "lock" icon in the file list. Additionally a lock owner (the user who locked the file) can manually unlock them via the "Locks" tab in the right sidebar. The "Locks" tab will only appear for files that have active locks.
 
+=== Foreign Keys in Database
+
+Please note that foreign keys have been added with the :xref:webdav-locks[WebDAV Locks] feature.
+This is the first time ownCloud implements foreign keys.
+
+==== How Does This Affect Each Database
+
+===== SQLite
+
+Foreign keys are, by default, disabled in SQLite.
```

Comment:
```suggestion
Foreign keys are, by default, disabled in SQLite. You must ensure that foreign keys are enabled in your SQLite installation.
```
Maybe say something like this?
And maybe also add some note (with link to the supported systems doco) to remind readers that SQLite is not recommended/supported for production use anyway.

Reply:
> remind readers that SQLite is not recommended/supported for production use anyway

👍